### PR TITLE
Adding Keplr Intereactions for Wallet and Handle an Edge Case

### PR DESCRIPTION
**Pull request by rabi-siddique**
The PR adds Kepler Interactions for:
- Creating a new Wallet using a Private Key. 
- Creating a new Wallet using an existing 24-word phrase. 

And it handles an edge case illustrated below:
Scenario:
Test Case 1: The user has to connect with the wallet. The user clicks the connect wallet button. Popups show and the user approves the connection. 
Test Case 2: The user is performing some operation. The user needs to connect to the wallet. The user clicks the connect wallet button. No popup shows ==> The user was already connected to the wallet as this happened in the first test case. 

The PR addresses this edge case and handles scenarios where the user is already connected to the wallet.

### CODE REVIEW

File: support/commands.js [hunk 1]
```diff
@@ -443,4 +443,12 @@ Cypress.Commands.add('switchToExtensionWindow', () => {
...
\ No newline at end of file
+});
+
+Cypress.Commands.add('makeOfferAndConfirmTransaction', () => {
+  cy.contains('agoric1p2aqakv3ulz4qfy2nut86j9gx0dx0yw09h96md');
+  cy.contains('Make an Offer').click();
+  return cy.confirmTransaction().then(taskCompleted => {
+    expect(taskCompleted).to.be.true;
+  });
+});
```
Comment (frazarshad) [inline support/commands.js:454]: these functions should not contain logic related to our offer up dapp since these will be part of the synpress framework and will be exposed to all users

File: commands/playwright-keplr.js [hunk 2]
```diff
@@ -361,7 +385,7 @@
         retries = 0;
         await page.bringToFront();
         await module.exports.waitUntilStable(page);
-        return page;
+        return page
```
[the remark this answers is not in the document]
Comment (frazarshad) [reply commands/playwright-keplr.js:388]: :point_up:
